ros2_ws/src/warehouse_robot/launch/launch_sim.launch.py

```diff
@@ -58,7 +58,7 @@ def generate_launch_description():
     diff_drive_spawner = Node(
         package="controller_manager",
         executable="spawner",
-        arguments=["diff_cont", "--controller-manager-timeout", "30"],
+        arguments=["diff_cont"], "--controller-manager-timeout", "30"],
```

Comment:
I think I copy-pasted that wrong, my fault
```suggestion
        arguments=["diff_cont", "--controller-manager-timeout", "30"],
```